jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/CustomRequestLogTest.java

```diff
@@ -675,13 +670,50 @@ public boolean handle(Request request, Response response, Callback callback)
         assertEquals(HttpStatus.BAD_REQUEST_400, response.getStatus());
         log = _logs.poll(5, TimeUnit.SECONDS);
         assertThat(log, is("/no/host ConnectionStatus: 400 X"));
+    }
 
-        getResponses("""
-            GET /abort HTTP/1.1
-            Host: localhost
+    @Test
+    public void testLogAbortConnectionStatus() throws Exception
+    {
+        start("%U ConnectionStatus: %s %X", new SimpleHandler()
+        {
+            @Override
+            public boolean handle(Request request, Response response, Callback callback)
+            {
+                Callback cbk = Callback.from(() -> callback.failed(new QuietException.Exception("test fail")), callback::failed);
+                Content.Sink.write(response, false, "data", cbk);
+                return true;
+            }
+        });
 
-            """, 1);
-        log = _logs.poll(5, TimeUnit.SECONDS);
+        try (Socket socket = new Socket("localhost", _serverConnector.getLocalPort()))
+        {
+            socket.setSoTimeout(10000);
+            socket.setTcpNoDelay(true);
+
+            OutputStream output = socket.getOutputStream();
+            output.write("""
+                GET /abort HTTP/1.1
+                Host: localhost
+                    
+                """.getBytes(StandardCharsets.ISO_8859_1));
+            output.flush();
+
+            BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream(), StandardCharsets.ISO_8859_1));
```

Comment:
Can we use `HttpTester` instead?

Reply:
No, because we want to test that no last chunk is sent.    I've added comments to that effect.